pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1258,6 +1264,10 @@ func PublishBlobInfoToZedCloud(ctx *zedagentContext, blobSha string, blobStatus
 			CreateTime:             createTime,
 		}
 		ReportBlobInfo.Resources = &info.ContentResources{CurSizeBytes: blobStatus.Size}
+		if !blobStatus.ErrorTime.IsZero() {
+			errInfo := encodeErrorInfo(blobStatus.ErrorAndTime())
+			ReportBlobInfo.Err = errInfo
+		}
```

Comment:
I guess nobody checked that the blob info contained the errors!

Reply:
ok